.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    macos:
+      xcode: "9.2.0"
+    shell: /bin/bash --login -eo pipefail
```

Comment:
This is necessary in combination with the `Set Ruby Version` to run a [non-standard variant of Ruby on macOS](https://circleci.com/docs/2.0/testing-ios/#custom-ruby-versions), which is outdated and will prevent us from using Danger:

```
Fetching gem metadata from https://rubygems.org/.............
public_suffix-3.0.1 requires ruby version >= 2.1, which is incompatible with the
current version, ruby 2.0.0p648
```